packages/react-portal/src/virtualParent/getParent.ts

```diff
@@ -0,0 +1,13 @@
+import { getVirtualParent } from './getVirtualParent';
+
+/**
+ * Gets the element which is the parent of a given element.
+ * If `allowVirtuaParents` is `true`, this method prefers the virtual parent over
```

Comment:
nit
```suggestion
 * If `allowVirtualParents` is `true`, this method prefers the virtual parent over
```

Reply:
Good catch, I've removed this option completely, there is no sense in using this option